src/modules/powerrename/lib/srwlock.h

```diff
@@ -0,0 +1,79 @@
+#pragma once
+#include "stdafx.h"
+
+// Wrapper around SRWLOCK
```

Comment:
For a future update, consider switching to standard std::shared_mutex instead of rolling your own code.

Reply:
Will do